src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -1743,6 +1742,15 @@ def test_sql_vnet_mgmt(self, resource_group, resource_group_location, server):
                      JMESPathCheck('resourceGroup', rg),
                      JMESPathCheck('virtualNetworkSubnetId', vnet_id_2)])
 
+        #test sql server vnet-rule update rule 2 with vnet 1 and ignore-missing-vnet-service-endpoint flag
+        self.cmd('sql server vnet-rule update --name {} -g {} --server {} --subnet {} -i'
```

Comment:
in order for this to be testing that `-i` actually gets update, wouldn't you need to either create the vnet-rule without `-i`, or update it to `false` here?